packages/pipeline-editor/src/utils.ts

```diff
@@ -52,4 +55,56 @@ export default class Utils {
 
     return template;
   }
+
+  /*
```

Comment:
If this is meant to be a doc comment its missing the `**` here. If it's not meant to be a doc comment then you should remove this blank comment line.